apiary.apib

```diff
@@ -3262,6 +3265,12 @@ Create a new milestone.
         + due_on: `2018-01-01` (string, required)
         + name: `Initial setup` (string, required)
         + responsible_user_id: `e1240972-6cfc-4549-b49c-edda7568cc48` (string, required)
+        + `invoicing_method`: `time_and_materials` (enum[string])
```

Comment:
Q: I noticed the other fields (i.e. responsible_user_id, name etc) are not encapsulated in backticks; why is this one?